CONTRIBUTING.md

```diff
@@ -19,8 +19,14 @@ That's excellent, thank you for collaborating!
 
 With the developer mode open localhost:3000 [http://localhost:3000](http://localhost:3000)
 
-# Pets Love API
+## Running the project using Docker 🐳
 
-If you want to set local back-end I suggest you go to the back-end repository
+1. Install Docker
+2. Install docker-compose
+3. Making a fork of the project and clone it
+4. Build image `docker-compose build`
+5. Create and run container `docker-compose --service-ports run frontend`

```

Comment:
Hi @chachan I try run the project with this command and not work. Terminal says it is missing a command or command does not exist and try with ``` docker-compose run frontend ``` and work perfectly 😃

I think after this change we can merge. 

Thank you 😃